sdks/go/pkg/beam/testing/ptest/ptest.go

```diff
@@ -16,6 +16,14 @@
 // Package ptest contains utilities for pipeline unit testing.
 package ptest
 
+// beam-playground:
+//   name: PTest
+//   description: Contains utilities for pipeline unit testing.
```

Comment:
Why do we need utilities? Do they test anything?

Reply:
Oh, the package doesn't really test anything. Remove tag.